app/src/main/java/com/rob729/quiethours/Fragments/NewProfileFragment.kt

```diff
@@ -217,13 +217,12 @@ class NewProfileFragment : Fragment() {
             c.add(Calendar.DAY_OF_YEAR, 7)
         }
 
-        val profileData = workDataOf(Pair("Profile_Name", profile.name))
-        val vibData = workDataOf(Pair("VibrateKey", profile.vibSwitch))
+        var etime = "End Time: " + profile.ehr.toString() + ":" + profile.emin.toString() + " hours"
```

Comment:
@coder2699 I have tested the app. It is working fine.
One small change. Don't define the string in this way. use "End Time: ${profile.ehr}" to define the string.

Reply:
Done